app/javascript/mastodon/features/notifications_v2/index.tsx

```diff
@@ -62,7 +63,7 @@ export const Notifications: React.FC<{
   multiColumn?: boolean;
 }> = ({ columnId, multiColumn }) => {
   const intl = useIntl();
-  const notifications = useAppSelector(selectNotificationGroups);
+  const notifications = useAppSelector(selectNotificationGroups, isEqual);
```

Comment:
Shouldn't `selectNotificationGroups` be memoized and only change when the input groups change? `isEqual` might be somewhat wasteful here. But it's still better than spurious re-renders.

Reply:
It should be memoized, but I wasn’t able to figure out why it isn’t.

Reply:
Oh, I think this may be because of `selectSettingsNotificationsExcludedTypes`. Will have a look.